pom.xml

```diff
@@ -219,7 +219,7 @@
         <jsonSchemaValidatorVersion>2.2.14</jsonSchemaValidatorVersion>
         <junitVersion>4.13.1</junitVersion>
         <jythonVersion>2.7.2b2</jythonVersion>
-        <jrubyVersion>9.2.14.0</jrubyVersion>
+        <jrubyVersion>9.4.2.0</jrubyVersion>
```

Comment:
why is this also needed in the context of a snakeyaml update?

Reply:
Jruby 9.2.14 uses snakesyaml 1.3.x I think and would bring in that transitively, hence upgrading that eventhough Im bit concerned about all the other changes since then in it

Reply:
Verify if the jenkins jobs for this branch are green. The dependency tree says that Jruby has snakeyaml as dependency?

Reply:
@fsgonz This is what I found, mule-module-scripting-jruby has a dependency on jruby-core and jruby-lib. Ran a dependency:tree on jruby lib 9.2.14.0 that we were usibg, it b rings in snakeyaml 1.26  due to
 org.jruby:jruby-stdlib:jar:9.2.14.0
    +- rubygems:psych:gem:3.2.0:provided
      \- org.yaml:snakeyaml:jar:1.26:provided

So we had to switch to newer snakeyaml 2.0 in mule-module-scripting-jruby . interestingly jruby has switched to snakeyaml engine to bypass vulenrability, so they dint inclkude snakeyaml.jar anymore but the engine in jruby-stdlib:jar:9.4.3.0-SNAPSHOT or 9.4.2.0 that I changed

+- rubygems:psych:gem:5.1.0:provided
[INFO] |  \- org.snakeyaml:snakeyaml-engine:jar:2.6:provided